common/execution_context.go

```diff
@@ -78,27 +78,29 @@ type ExecutionContext struct {
 
 // NewExecutionContext creates a new JS execution context.
 func NewExecutionContext(
-	ctx context.Context, session *Session, frame *Frame,
-	id runtime.ExecutionContextID, logger *Logger,
+	ctx context.Context,
+	s session,
+	f *Frame,
+	id runtime.ExecutionContextID,
+	l *Logger,
```

Comment:
One arg per line seems excessive. The previous version looked readable to me.

I know this is a style issue, but I don't see what we gain from it, and it just pads LOCs.

Reply:
I find the previous one harder to read :( Definitely a style issue :)